### DISCUSSION TITLE
Portal Engine Error On Local Windows Pimcore

### DISCUSSION THREAD

**Discussion by KasthuriAnandhan** [pimcore/pimcore#17239] [Q&A / Support]
Hi Guys @kingjia90, @fashxp ,

I was installed skeleton pimcore on my windows system and used our enterprise token on my local, and successfully created a portal page with data object and asset pool, but when I navigate to product or asset pool facing the following error. anybody can help me to fix this. and I was installed elastic search also will share it details also.

Portal Page Error
![image](https://github.com/pimcore/pimcore/assets/125338676/154f0847-db7d-4ee9-a00c-25af9c1ffb2c)

Elastic Search Details
![image](https://github.com/pimcore/pimcore/assets/125338676/6fbac33f-e3f4-4d0b-a49b-86f601f0dd14)
![image](https://github.com/pimcore/pimcore/assets/125338676/ffb86df5-ec01-4a3d-a08f-fa14e97b1e17)

**Comment by fashxp** [comment]
Hi, first thing that pops into my eye: when using latest version of portal engine, you need to use opensearch insead of elasticsearch. 

Cheers...

**Comment by KasthuriAnandhan** [comment]
Hi @fashxp 
As you said I was removed elastic search service on my local machine and installed open search.
![image](https://github.com/pimcore/pimcore/assets/125338676/4a27d53b-e71f-412d-9288-77b10160b5f5)

![image](https://github.com/pimcore/pimcore/assets/125338676/d4e124e3-2ae8-4c22-851a-1d1533fa7346)

Configured also in config.yaml file

![image](https://github.com/pimcore/pimcore/assets/125338676/e9a3a28b-fd00-4dff-a467-28fa319b3b54)

Now the another error occurs, shown in below
![image](https://github.com/pimcore/pimcore/assets/125338676/329dfa30-0ea2-4dba-b741-aeb2301fbbae)

My password has "@" this may be any problem? if I didn't configured anything in the config.yaml file another error occurs like below

![image](https://github.com/pimcore/pimcore/assets/125338676/7e69c326-a92b-4324-ba02-d8182a3c7d7a)

Can you please help me to resolve this issue, and able to access the data object and asset pools successfully.

**Comment by fashxp** [comment]
No sure if your setup is set up to use parameters as opensearch configuration? Pimcore doesn't use these parameters by default. See https://pimcore.com/docs/platform/Generic_Data_Index/Configuration/Opensearch for details on how to setup the opensearch configuration. 

Concerning the @ ... you probably need to escape that somehow, that is a symfony topic, not sure how this can be done...
